src/components/atoms/Footer-item.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
```

Comment:
this wasnt what was expected only Footer-Items have to be exported in the form of a simple array later to be used inside the Footer component inside organisms to have clean code easy to maintain debug and scale

Reply:
I have edited the FooterItem but I don't know if it is as you described?